src/api/telegram-subscription/routes/telegram-subscription.ts

```diff
@@ -4,4 +4,49 @@
 
 import { factories } from '@strapi/strapi';
 
-export default factories.createCoreRouter('api::telegram-subscription.telegram-subscription');
+const defaultRouter = factories.createCoreRouter('api::telegram-subscription.telegram-subscription');
+
+const customRouter = (innerRouter, extraRoutes = []) => {
+  let routes;
+  return {
+    get prefix() {
+      return innerRouter.prefix;
+    },
+    get routes() {
+      if (!routes) routes = innerRouter.routes.concat(extraRoutes);
+      return routes;
+    },
+  };
+};
+
+const myExtraRoutes = [
+  {
+    method: 'GET',
+    path: '/tg-subscriptions/:account',
```

Comment:
the name is a bit weird. Why not something like:
- `/accounts/{account}/subscriptions?channel=telegram`

Reply:
Fixed